xlint

```diff
@@ -251,11 +251,13 @@ for template; do
 		done
 	fi
 
-	: ${LICENSE_LIST:=/usr/share/spdx/license.lst}
+	: "${LICENSE_LIST:=/usr/share/spdx/license.lst}"
 	if [ -f "${LICENSE_LIST}" ]; then
 		sed -n 's/license="\(.*\)"/\1/p' "$template" | tr , "\n" | while read -r l; do
 			if echo "$l" | grep -q 'custom:'; then
 				continue
+			elif echo "$l" | grep -qi '^public\s*domain$'; then
```

Comment:
Don't like the variable space here.

And we'd need to standardize. Perhaps `Public Domain`.

Reply:
then I could just add to the license list and fixup in void-packages.

Reply:
oh whoops that is not right.